yarn.lock

```diff
@@ -20144,9 +20144,9 @@ ursa-optional@~0.10.0:
     bindings "^1.5.0"
     nan "^2.14.2"
 
-"use-inside@git+https://github.com/aragon/use-inside.git#16f321e499d8f89e9b0fb0999e2d21db2b119bbf":
+"use-inside@https://github.com/aragon/use-inside#16f321e499d8f89e9b0fb0999e2d21db2b119bbf":
```

Comment:
wondering why the lock changed with no new package installed?

Reply:
I don’t really know why this was updated. I always run yarn and delete node_modules.

Reply:
it is because in the latest version of yarn, they started supporting the installing of npm modules from github repository directly using `https://....` URLs. Earlier they used to support the old way `git+ssh://....` or `git+https://...`.

This should have been fixed in the 1hive-ui lib itself but then it wasn't for some reason. Will fix that when I am working on the updating of the migration to peerDeps (reducing the size of the library.